src/chunkserver/op_request.cpp

```diff
@@ -264,21 +262,8 @@ void ReadChunkRequest::Process() {
         /*
          * 将read扔给并发层出于两个原因：
          *  (1). 将read I/O操作和write等其它I/O操作都放在并发层处理，以便隔离
-         *  disk I/O和其他逻辑
-         *  (2). 为了保证线性一致性read的语义。因为当前apply是并发的，所以applied
-         *  index更新也是并发的，尽管applied index更新能够保证单调的，但是可能会存
-         *  在更新跳跃的情况，例如，index=6,7的2个op同时进入并发模块，并且都执行成
-         *  功返回了，这个时候leader挂了，new leader选出来，new leader上面有
-         *  index=6,7两个op的日志，但是没有apply，那么new leader必然需要回放这两
-         *  条日志，因为是并发的，所以index=7的op log可能先于index=6的被apply，然后
-         *  new leader的applied index会被更新为7，这个时候client来了一个想读index=6
-         *  的op写下的数据，携带的是applied index=7，这个时候ChunkServer比较携带的
-         *  applied index和Chunkserver的applied index，那么会判定通过走直接读，但是
-         *  ChunkServer实际上index=6的数据还没落盘。那么就会出现stale read。解决方法
-         *  就是read也进并发层排队，那么需要read index=6的read request，必定会排在
-         *  index=6的op的后面，也就是它们操作的是同一个chunk，并发层会将它们放在同一个
-         *  队列中，这样就能保证index=6的op apply之后，read才会被执行，这样就不会出现
-         *  stale read，保证了read的线性一致性
+         *  disk I/O和其他逻辑。
+         *  (2). 为了保证线性一致性read的语义。
```

Comment:
Should we add a comment in CopysetNode::on_leader_start to explain the ordering?